Azure/sonic-buildimage
# [Mux] Do not clean-up HW_MUX_CABLE_TABLE from State DB

#### Why I did it
By removing the dependency of pmon from swss by this [PR ](https://github.com/Azure/sonic-buildimage/pull/7614), restarting swss should not clean-up HW_MUX_CABLE_TABLE.

#### How I did it
Fix the clean-up script

#### How to verify it
After populating the HW_MUX_CABLE_TABLE in State DB, restart swss. Verify that MUX_CABLE_TABLE is written with "unknown" state.

```
127.0.0.1:6379[6]> HGETALL "HW_MUX_CABLE_TABLE|Ethernet4"
1) "state"
2) "standby"

127.0.0.1:6379[6]> HGETALL "MUX_CABLE_TABLE|Ethernet4"
1) "state"
2) "standby"
```

Restart SWSS:

$ sudo systemctl restart swss

```
127.0.0.1:6379[6]> HGETALL "MUX_CABLE_TABLE|Ethernet4"
1) "state"
2) "unknown"
```




#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
